contributing/README.md

```diff
@@ -10,6 +10,97 @@ us on [Slack]. Please also take a look at our [code of conduct], which details
 how contributors are expected to conduct themselves as part of the Crossplane
 community.
 
+## Getting Started Contributing to Crossplane
+
+There is always lots of exciting work going on in the Crossplane project and
+there are many places where new contributors can start to get more involved.
+This section will help you understand the various opportunities across the
+community and where you can start contributing.
+
+To see all of the ways to connect with the rest of the community (community
+meetings, special interest groups, Slack discussions, etc.), please refer to the
+[Get Involved] section of the main README.
+
+The CNCF also maintains a very helpful [contributor
+guide][cncf-contributor-guide] that gives an excellent overview on how to get
+into open source in general and start making your first contributions. We highly
+recommend browsing this guide to kick start your contributor journey.
+
+### Crossplane Core Contributions
+
+The roadmap for improvements, fixes, and work items for Crossplane's [core] and
+[runtime] are triaged and tracked in a single [GitHub project]. Here are a few
+useful views of that project to help new contributors find areas where help is
+needed:
+
+* Issues that are useful for the project, but have lower complexity, are a great
+  fit for folks just beginning to get involved in the code base. These issues
+  are identified by the `good first issue` label.
+  * [Good first issues]
+* High priority work items are marked as P0/P1. These issues tend to have the
+  biggest impact on the project, but also tend to be complex and challenging.
+  * [P0/P1 issues with no owner]
+  * [All P0/P1 issues]
+
+### Extensions Ecosystem Contributions
+
+Now that the core functionality of Crossplane is considered fairly mature and
+stable, the vibrant ecosystem of Crossplane extensions has the most
+opportunities for contributors to get involved.  There is a wide range of
+Providers and Functions that could use your help to continue maturing and
+delighting the community.
+
+* Providers
+  * If a Provider you need doesn't exist yet, you can create a [new provider] to
+    fill this gap in the ecosystem.
+  * The [Upjet framework] has become very popular for creating and maintaining
+    Providers, and has lots of functionality to invest in.
+    * The special interested group for Upjet is a great place to meet and
```

Comment:
```suggestion
    * The special interest group for Upjet is a great place to meet and
```

Reply:
hey, they are interested in Upjet too! 😂 - good catch, thanks @phisco!